docs/source/conf.py

```diff
@@ -114,13 +114,21 @@ def previous_version(ver):
 
 # Inserted at the bottom of all rst files.
 # Use for variables substitutions
+
+if tags.has('enterprise'):
+    print "Building BWC docs"
```

Comment:
yes, indeed: visible during build. Else I need to look at results, and if I'm on travis, or build003, good to check